minorminer/utils/parallel_embeddings.py

```diff
@@ -128,7 +128,7 @@ def find_multiple_embeddings(
         S: The source graph to embed.
         T: The target graph in which to embed.
         max_num_emb: Maximum number of embeddings to find.
-            Defaults to 1, set to float('Inf') to find the maximum possible
+            Defaults to 1, set to None to find the maximum possible
```

Comment:
This is actually a breaking change. For backwards compatibility, consider supporting `inf` as well, but throwing a deprecation warning if called with it. (I haven't checked, but if `Inf` support hasn't been released yet, than no need to deprecate it, obviously.)

Reply:
Indeed, this code hasn't been released yet, so please ignore the comment above.

Reply:
The first pull request was merged prematurely. I think @thisac wants to take a second look at this tidy up request as well.